options/locale/locale_en-US.ini

```
@@ -1200,6 +1205,7 @@ pulls.reject_count_1 = "%d change request"
 pulls.reject_count_n = "%d change requests"
 pulls.waiting_count_1 = "%d waiting review"
 pulls.waiting_count_n = "%d waiting reviews"
+pulls.wrong_commit_id = "commit id must be a commit id on the base branch"
```

So yes it should be target, sorry.

```suggestion
pulls.wrong_commit_id = "commit id must be a commit id on the target branch"
```